frontend/src/services/apiService.js

```diff
@@ -4,7 +4,7 @@ export const fetchWrapper = async (url, options = {}) => {
   const controller = new AbortController();
   const signal = controller.signal;
   try {
-    const response = await fetch(`${backendUrl}/${url}`, {
```

Comment:
I'd prefer to keep the backendUrl here, since this function is only used to query the backend